addons/binding/org.openhab.binding.icloud/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="icloud"
+				 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				 xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+				 xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+	<name>iCloud Binding</name>

```

Comment:
Somehow indention is messed up here. Please use the functionality of your editor to replace all tabs by for 4 spaces, then [apply](https://www.eclipse.org/pdt/help/html/formatting_code.htm) the auto formatter functionality. You should do that for all files related to your binding.

@martinvw still don't get why this is not a project properties default!? Are you afraid of automatic changes to other code parts? I think there are appropriate settings to prevent that. Imho someone should auto-format the whole repository once at some point in time.